### ISSUE TITLE
"code --status" shows '|' as first character of the Workspace Stats

### ISSUE THREAD

**Issue by weinand** [microsoft/vscode#39777]
testing #39376:

"code --status" shows '|' as first character of the Workspace Stats:

![2017-12-06_11-17-16](https://user-images.githubusercontent.com/1898161/33657505-dca26fe2-da79-11e7-856a-2480b73f89f9.png)

Why do we need this?
If the reason is that otherwise the indentation doesn't align, I wonder why we do not need this for the process tree (where I've never seen indentation issues).

**Comment by bpasero** [comment]
@weinand this was added by me. I tried for 1 hour to not need it but without using it I am randomly seeing a line moving to the beginning.

To demonstrate how broken terminal rendering is (on macOS at least), see the following video where I execute the same command 5 times without using the | symbol as separator. You see how the last invocation is fucked up:

![flicker_chrome58](https://user-images.githubusercontent.com/900690/33659447-851f19b2-da80-11e7-9513-cc0582df866e.gif)

I am open for ideas how to resolve this...
